components/dashboard/src/components/start-workspace.tsx

```diff
@@ -354,6 +354,13 @@ export class StartWorkspace extends React.Component<StartWorkspaceProps, StartWo
         if (isError) {
             message = <div className="message action">
                 <Button className='button' variant='outlined' color='secondary' onClick={() => this.redirectToDashboard()}>Go to Workspaces</Button>
+                <Button className='button' variant='outlined' color='secondary' onClick={() => {
+                  if (this.workspace && this.state && this.state.workspaceInstance) {
+                    console.log(this.workspace.baseImageNameResolved);
+                    this.workspace.baseImageNameResolved = "gitpod/workspace-full";
```

Comment:
We'd need to pass an argument to the server API, telling it to ignore the image configuration and use the default.